### PR TITLE
Add support to build release candidates

### PR DESCRIPTION
**Pull request by mctavish**
This repository has previously been a mirror of a Google-internal one, and releases have been built from the internal repository.

This PR adds support to build release candidates directly from this repository than can be tested and uploaded to pypi.

Many unit tests have been added, but some are disabled until they can successfully pass on all supported versions of Python.

One change has been lumped into this: SetConditionalBreakpoint has been split into CreateConditionalBreakpoint and ActivateConditionalBreakpoint.  This change fixes an edge cases where a breakpoint could be created and not deleted if there is a failure at the wrong time.  This issue was discovered during testing.

### CODE REVIEW

**Review by jasonborg — APPROVED** [review]
It all looks good/reasonable

**Comment by mctavish** [comment]
Added tests!  PTAL

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+six
```
[the remark this answers is not in the document]
Comment (mctavish) [reply requirements.txt:1]: https://github.com/GoogleCloudPlatform/cloud-debug-python/issues/30 will track this cleanup.

**Review by jasonborg — APPROVED** [review]
Everything looks good.

File: tests/imphook2_test.py [hunk 2]
```diff
@@ -0,0 +1,521 @@
+"""Unit test for imphook2 module."""
```
[the remark this answers is not in the document]
Comment (mctavish) [reply tests/imphook2_test.py:1]: Hah.  Entirely valid.

**Pull request merged**